downloads/platform.md

```diff
@@ -76,9 +76,16 @@ You can uninstall Julia by deleting Julia.app and the packages directory in `~/.
 
 ## Linux and FreeBSD
 
-It is strongly recommended that the official generic binaries from the downloads page be used to install Julia on Linux and FreeBSD.
+It is strongly recommended that the official generic binaries from the downloads page be used to install Julia on Linux and FreeBSD. The following
+command downloads the latest Julia version and makes a symlink to allow the `julia` command to find the binary:
 
+```
+wget https://julialang-s3.julialang.org/bin/linux/x64/1.4/julia-{{ stable_release }}-linux-x86_64.tar.gz
```

Comment:
`1.4` here should be replaced with some placeholder

Reply:
`{{ stable_release_short }}`